src/coreclr/inc/eventtracebase.h

```diff
@@ -1153,6 +1153,19 @@ namespace ETW
             } ContentionFlags;
         } ContentionStructs;
     };
+
+    class WaitHandleLog
+    {
+    public:
+        typedef union _WaitHandleStructs
+        {
+            typedef  enum _WaitReason {
+                Unknown=0,
+                MonitorWait=1
+            } WaitReason;
```

Comment:
Maybe this also be called WaitSource like elsewhere, or all of them could be called WaitReason